services/src/main/java/org/keycloak/authentication/forms/RegistrationRecaptcha.java

```diff
@@ -214,50 +292,17 @@ public FormAction create(KeycloakSession session) {
 
     @Override
     public void init(Config.Scope config) {
-
     }
 
     @Override
     public void postInit(KeycloakSessionFactory factory) {
-
     }
 
     @Override
     public String getId() {
         return PROVIDER_ID;
     }
 
-    @Override
-    public String getHelpText() {
-        return "Adds Google Recaptcha button.  Recaptchas verify that the entity that is registering is a human.  This can only be used on the internet and must be configured after you add it.";
-    }
-
-    private static final List<ProviderConfigProperty> CONFIG_PROPERTIES = new ArrayList<ProviderConfigProperty>();
-
-    static {
-        ProviderConfigProperty property;
-        property = new ProviderConfigProperty();
-        property.setName(SITE_KEY);
-        property.setLabel("Recaptcha Site Key");
-        property.setType(ProviderConfigProperty.STRING_TYPE);
-        property.setHelpText("Google Recaptcha Site Key");
-        CONFIG_PROPERTIES.add(property);
-        property = new ProviderConfigProperty();
-        property.setName(SITE_SECRET);
-        property.setLabel("Recaptcha Secret");
-        property.setType(ProviderConfigProperty.STRING_TYPE);
-        property.setHelpText("Google Recaptcha Secret");
-        CONFIG_PROPERTIES.add(property);
-
-        property = new ProviderConfigProperty();
-        property.setName(USE_RECAPTCHA_NET);
```

Comment:
Could you elaborate on why we are dropping support for using `recaptcha.net` as an alternative when there are constraints on using the `google.com` domain?

Reply:
Good catch! This was a misunderstanding on my part, I added this option back and updated the doc accordingly. Sorry for the mishap.